### ISSUE TITLE
Add the ability to add/remove users from conversations

### ISSUE THREAD

**Issue by vocatwk** [vocatwk/CodeU-Nemo#44]

**Comment by lealgamb** [comment]
Hey Veronica, I was thinking that this feature is similar to the search bar. Do you think we should work on a class that does all the search functionality. Then we can use objects of this class for places like adding users and the search bar.

**Comment by vocatwk** [comment]
Sure, but what do you think this class would have though? cause I was just thinking a method that searches for users based on id(which we might already have) would be enough for our brute force solution.

**Comment by lealgamb** [comment]
Yeah it might not be very code heavy, but if we have similar features then it can help us prevent repetitive code. It also doesn't have to be a Java class but a jsp, like the searchbar one. Also, how are you planning to search based on id? From a user perspective it might be better to search by username.

**Comment by yebrahim** [comment]
I really like the thinking to reuse code for other features, this will take us a long way if we do it right.
Don't we need support for conversation access control first though? Otherwise adding/removing users is meaningless.

**Comment by vocatwk** [comment]
Yes, I was thinking of doing that before implementing this. @lealgamb Sorry, I meant I'd search based on username. I was thinking I would have a list of usernames associated with each conversation.

**Comment by vocatwk** [comment]
I am currently working on this. I was thinking any member of that conversation can add/remove any other user to start with with the user who creates the conversation automatically becoming the first member. Later, I can set up an admin for the conversation and control who can add/remove users from the conversations if you guys think it's worth it.
